scene/gui/base_button.cpp

```diff
@@ -145,9 +150,10 @@ void BaseButton::_toggled(bool p_pressed) {
 }
 
 void BaseButton::on_action_event(Ref<InputEvent> p_event) {
-	if (p_event->is_pressed()) {
+	if (p_event->is_pressed() && !status.pressed_down_with_focus) {
```

Comment:
```suggestion
	if (!status.pressed_down_with_focus && p_event->is_pressed()) {
```
Cheaper check first. Same below.